charts/greenhouse/templates/plugin-perses.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: greenhouse.sap/v1alpha1
+kind: Plugin
+metadata:
+  name: perses
+spec:
+  disabled: false
+  displayName: perses
+  optionValues:
+    - name: perses.ingress.annotations
+      value:
+        disco: "true"
+        kubernetes.io/tls-acme: "true"
+    - name: perses.ingress.enabled
+      value: true
+    - name: perses.ingress.hosts
+      value:
+        - host: {{ index .Values.perses.ingress.hosts 0 "host" }}
+          paths:
+            - path: /
+              pathType: Prefix
+    - name: perses.ingress.ingressClassName
+      value: nginx
+    - name: perses.ingress.tls
+      value:
+        - hosts:
+            - {{ index .Values.perses.ingress.tls.hosts 0 }}
+          secretName: {{ printf "%s-%s" "tls" (first .Values.perses.ingress.hosts).host | replace "." "-" }}
```

Comment:
Please have a look at this https://github.com/cloudoperators/greenhouse/blob/6564bc562f1dc56e20d1890c9a3ebf25013ea7dc/charts/idproxy/templates/ingress.yaml#L28-L33
I tried to reduce the amount of values required to be set when deploying Greenhouse itself, and opted to hard-code the subdomains within helm templates. This way it is sufficient to have the one dns domain for Greenhouse and everything else just has to add subdomains.

Reply:
Cool. I think for the most part we can get away with some defaults for the ingress hosts. In the end the subdomain can also be made configurable, if the need arises at some point. But I think for now it should be sufficient. Thanks for the changes.